packages/core/test/slice-test.js

```diff
@@ -116,6 +126,11 @@ describe('slice', function () {
   })
 
   describe('skipAfter', function () {
+    it('given canonical empty stream, should return canonical empty', () => {
+      const s = skipAfter(_ => false, empty())
```

Comment:
Since we don’t care whether `f` returns `true` or `false`, I think it would be clearer to use `Boolean`:

```ts
takeWhile(Boolean, empty())
```